### DISCUSSION TITLE
[Feature Request]: drop-in replacement for the openai embeddings API

### DISCUSSION THREAD

**Discussion by nnWhisperer** [qdrant/fastembed#185] [Ideas]
### What happened?

How can we run fastembed as a drop-in replacement for the openai embeddings API? Do you have a suggestion or known ways(fastembed+another library that serves the fastembed) to use? Thank you

### Version

0.2.4 (Latest)

**Comment by NirantK** [comment]
Not exactly a drop in replacement, but definitely a REST: https://github.com/michaelfeil/infinity
